packages/browser/src/diagnose-sdk.ts

```diff
@@ -0,0 +1,34 @@
+import { getCurrentScope } from '@sentry/core';
+
+/**
+ * A function to diagnose why the SDK might not be successfully sending data.
+ *
+ * Possible return values wrapped in a Promise:
+ * - `"no-client-active"` - There was no active client when the function was called. This possibly means that the SDK was not initialized yet.
+ * - `"sentry-unreachable"` - The Sentry SaaS servers were not reachable. This likely means that there is an ad blocker active on the page or that there are other connection issues.
+ *
```

Comment:
should we also handle e.g. 500 errors here? e.g. `sentry-down` or something like this?

Reply:
No I don't think so. This is purely for connection issues.

Reply:
That seems reasonable